### PR TITLE
4.61.0 changelog

### PR DESCRIPTION
**Pull request by pmaytak**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,33 +1,44 @@
...
+- Removed support for UWP. MSAL.NET packages will no longer include `uap10.0.17763` binary. Existing applications should migrate to modern frameworks like WinUI 3. See [4717](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4717) and [Announcing the Upcoming Deprecation of MSAL.NET for Xamarin and UWP](https://devblogs.microsoft.com/identity/uwp-xamarin-msal-net-deprecation/).
+- Removed Windows Forms dependency from the main package, Microsoft.Identity.Client, which will no longer include `net6.0-windows7.0` binary. Existing desktop applications targeting `net6.0-windows` should reference Microsoft.Identity.Client.Broker when using interactive authentication with Windows Broker and reference Microsoft.Identity.Client.Desktop when authenticating with browser. See [4468](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4468).
```
Comment (gladjohn) [inline CHANGELOG.md:6]:
```suggestion
- Removed Windows Forms dependency from `Microsoft.Identity.Client`, which will no longer include `net6.0-windows7.0` binary. Existing desktop applications targeting `net6.0-windows` should reference `Microsoft.Identity.Client.Broker` when using interactive authentication with Windows Broker and reference `Microsoft.Identity.Client.Desktop` when authenticating with browser. See [4468](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4468).
```

**Review by gladjohn — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,33 +1,44 @@
...
+- Removed support for UWP. MSAL.NET packages will no longer include `uap10.0.17763` binary. Existing applications should migrate to modern frameworks like WinUI 3. See [4717](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4717) and [Announcing the Upcoming Deprecation of MSAL.NET for Xamarin and UWP](https://devblogs.microsoft.com/identity/uwp-xamarin-msal-net-deprecation/).
+- Removed Windows Forms dependency from `Microsoft.Identity.Client`, which will no longer include `net6.0-windows7.0` binary. Existing desktop applications targeting `net6.0-windows` should reference `Microsoft.Identity.Client.Broker` when using interactive authentication with Windows Broker and reference `Microsoft.Identity.Client.Desktop` when authenticating with browser. See [4468](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4468).
```
Comment (pmaytak) [inline CHANGELOG.md:6]:
```suggestion
- Removed Windows Forms dependency from `Microsoft.Identity.Client`, which will no longer include `net6.0-windows7.0` binary. Existing desktop applications targeting `net6.0-windows` should reference `Microsoft.Identity.Client.Broker` when using interactive authentication with Windows Broker and call `WithBroker(BrokerOptions)`; or reference `Microsoft.Identity.Client.Desktop` when authenticating with browser and call `WithWindowsEmbeddedBrowserSupport()`. There are no changes to the usage of system browser. See [4468](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4468).
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,33 +1,44 @@
...
+- Removed support for UWP. MSAL.NET packages will no longer include `uap10.0.17763` binary. Existing applications should migrate to modern frameworks like WinUI 3. See [4717](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4717) and [Announcing the Upcoming Deprecation of MSAL.NET for Xamarin and UWP](https://devblogs.microsoft.com/identity/uwp-xamarin-msal-net-deprecation/).
+- Removed Windows Forms dependency from `Microsoft.Identity.Client`, which will no longer include `net6.0-windows7.0` binary. Existing desktop applications targeting `net6.0-windows` should reference `Microsoft.Identity.Client.Broker` when using interactive authentication with Windows Broker and call `WithBroker(BrokerOptions)`; or reference `Microsoft.Identity.Client.Desktop` when authenticating with browser and call `WithWindowsEmbeddedBrowserSupport()`. There are no changes to the usage of the system browser. See [4468](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4468).
```
Comment (pmaytak) [inline CHANGELOG.md:6]:
```suggestion
- Removed Windows Forms dependency from `Microsoft.Identity.Client`, which will no longer include `net6.0-windows7.0` binary. Existing desktop applications targeting `net6.0-windows` should reference `Microsoft.Identity.Client.Broker` when using [interactive authentication with Windows Broker](https://aka.ms/msal-net-wam) and call `WithBroker(BrokerOptions)`; or reference `Microsoft.Identity.Client.Desktop` when [authenticating with browser](https://aka.ms/msal-net-uses-web-browser) and call `WithWindowsEmbeddedBrowserSupport()`. There are no changes to the usage of the system browser. See [4468](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4468).
```

**Review by gladjohn — APPROVED** [review]

**Comment by pmaytak** [comment]
Docs updates: https://github.com/MicrosoftDocs/microsoft-authentication-library-dotnet/pull/406

https://review.learn.microsoft.com/en-us/entra/msal/dotnet/acquiring-tokens/desktop-mobile/wam?branch=pmaytak%2Fnet6-win#package-availability

https://review.learn.microsoft.com/en-us/entra/msal/dotnet/acquiring-tokens/using-web-browsers?branch=pmaytak%2Fnet6-win#browser-availability

**Pull request merged**